core/domain/opportunity_services.py

```diff
@@ -86,12 +87,30 @@ def get_exploration_opportunity_summary_from_model(model):
     new_incomplete_translation_language_codes = (
         model.incomplete_translation_language_codes + missing_language_codes)
 
+    translation_in_review_counts = {}
+    for language_code in constants.SUPPORTED_CONTENT_LANGUAGES:
+        suggestions_in_review = (
+            suggestion_services
+            .get_translation_suggestions_in_review_by_exploration(
+                model.id,
+                language_code['code']
+            ))
+        filtered_suggestions = []
+        for suggestion in suggestions_in_review:
+            if suggestion is not None:
+                filtered_suggestions.append(suggestion)
```

Comment:
If all you're doing is fetching the count, then perhaps you can just increment a counter here rather than storing a full list of filtered suggestions which you subsequently discard?

Even better, see if you can structure things such that the count can be fetched with a get-multi or keys-only call to the datastore (see e.g. https://stackoverflow.com/questions/11596869/is-it-best-to-query-by-keys-only-true-then-get-multi-or-just-full-query). You might need to do a bit of experimenting to see which approach is fastest; please document your experiments and results in this PR thread.

Reply:
Done